parlai/tasks/cornell_movie/agents.py

```diff
@@ -4,109 +4,71 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-from parlai.core.teachers import FbDeprecatedDialogTeacher
+from parlai.core.teachers import DialogTeacher
 from .build import build
 from parlai.utils.data import DatatypeHelper
 
 import copy
 import os
+import codecs
 
 
-def _path(opt, filtered):
-    # Build the data if it doesn't exist.
-    build(opt)
-    dt = opt['datatype'].split(':')[0]
-    return os.path.join(opt['datapath'], 'CornellMovie', dt + filtered + '.txt')
+def _path(opt, *additions):
+    return os.path.join(opt['datapath'], 'CornellMovie', *additions)
```

Comment:
Indeed, it seems like `build(opt)` was lost in the change.

Reply:
Oh good catch, thank you!